app/models/instance.rb

```diff
@@ -6,6 +8,13 @@ def default
       result
     end
 
+    # Returns the current instance from ActsAsTenant
+    #
+    # @return [Instance] current instance
+    def current
```

Comment:
It will be useful for me to get the instance settings object in controller when do views : <code> Instance.current.settings </code>